src/implementations/vanilla/components/page-header/page-header-ec.scss

```diff
@@ -169,7 +170,7 @@
 }
 
 .ecl-page-header--overlay-dark {
-  .ecl-page-header__background::before {
+  .ecl-page-header__background-container::before {
     background-color: rgba(map.get(theme.$color, 'black'), 0.5);
```

Comment:
It seems we could set this overlay effect with a single css property, mix-blend-mode, https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode, instead of using a pseudo element absolutely positioned, it's not supported in internet explorer, though, in case we are still bothered by that.
Also relevant for the banner pr, this property has been used for the "image-overlay" variant

Reply:
I tried it, but without success here.
As there is a blue background on the whole element, the overlay ended up being blue too

Reply:
@planctus I confirm that mix-blend-mode does not work as we would like to.
On the banner too, the result is not correct. It takes into consideration all the background set on the element, so the end result is something odd and depending on the background colors (this is clearly visible on the EU banner in your PR)

Reply:
ok, i will change this in the banner as well..